sys/net/sock/async/event/sock_async_event.c

```diff
@@ -98,7 +98,7 @@ void sock_tcp_queue_event_init(sock_tcp_queue_t *queue,
                                event_queue_t *ev_queue,
                                sock_tcp_queue_cb_t handler)
 {
-    sock_async_ctx_t *ctx = sock_tcp_queue_get_async_ctx(sock);
+    sock_async_ctx_t *ctx = sock_tcp_queue_get_async_ctx(queue);
```

Comment:
Found this bug while trying to integrate `sock_async` with `sock_tcp`. Hope nobody minds I piggy-back this bug fix here. It is in a separate commit, so I can always factor it out if not.